include/seqan3/std/algorithm

```diff
@@ -93,5 +94,10 @@ using SEQAN3_DOXYGEN_ONLY(move =) ::ranges::move;
 */
 using SEQAN3_DOXYGEN_ONLY(sort =) ::ranges::sort;
 
+/*!\typedef std::ranges::transform
+* \brief Alias for ranges::transform. Sorts a range.
```

Comment:
I don't understand the part `Sorts a range.`:
That's not what we expect from the transform function...

Reply:
Oh I missed that